pkgs/by-name/si/sioyek/package.nix

```
@@ -74,12 +74,13 @@ stdenv.mkDerivation (finalAttrs: {
         installManPage resources/sioyek.1
       '';
 
-  passthru.updateScript = nix-update-script {
-    extraArgs = [
-      "--version"
-      "branch=development"
-    ];
-  };
+  # keep disabled until https://github.com/Mic92/nix-update/issues/321 is fixed
+  # passthru.updateScript = nix-update-script {
+  #   extraArgs = [
+  #     "--version"
+  #     "branch=development"
+  #   ];
+  # };
```

Could something like this be an alternative?

```suggestion
  passthru.updateScript = unstableGitUpdater {
    branch = "development"
  };
```

---

Thanks for the tip, didn't know this existed. 

https://github.com/NixOS/nixpkgs/blob/0d534853a55b5d02a4ababa1d71921ce8f0aee4c/pkgs/common-updater/unstable-updater.nix#L10-L24

For those trying to test locally:

```bash
nix-shell -I maintainers/scripts/update.nix --argstr package sioyek
```

```text
this derivation will be built:
  /nix/store/wcrpnkvid769swfyhhbrmmpjkh3yc5pq-packages.json.drv
building '/nix/store/wcrpnkvid769swfyhhbrmmpjkh3yc5pq-packages.json.drv'...

Going to be running update for following packages:
 - sioyek-2.0.0-unstable-2025-03-08

Press Enter key to continue...

Running update for:
 - sioyek-2.0.0-unstable-2025-03-08: UPDATING ...
 - sioyek-2.0.0-unstable-2025-03-08: DONE.

Packages updated!
```

Not sure if I should set `tagFormat`, seems relatively few packages do this and it works without it.

I assume like `nix-update` this still works with the merge bot?